charts/argo-cd/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: v2
 appVersion: v2.5.0
+kubeVersion: ">=1.22.0"
```

Comment:
Added info about minimum version of K8s cluster

Reply:
Hmmm, I am unsure wether we should add this constraint or not. Redis-ha is an optional feature

Reply:
Yes but `seccomp` profile is 1.22 feature.

Reply:
This will exclude many users like me who don't use redid-ha feature and run k8s 1.21.

Reply:
I am unsure what path we should follow. Upstream 2.5 manifests (kustomize) also set seccomp and therefore ensures that customers run k8s >= 1.22

Maybe we should follow that also. Otherwise our charts supports a scenario which is not supported from the project itself.

Reply:
@mubarak-j Not really - Azure and GKE already allow 1.22 as a minimum. For AWS the EOL is February 2023.